src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -382,10 +395,19 @@ def run_bwa(job, job_vars):
         return_input_paths(job, work_dir, ids, 'ref.fa.alt')
 
     if input_args['use_bwakit']:
+
+        # add a read group line
+        # this is an illumina only pipeline, therefore, hardcoding illumina is OK
+        uuid = input_args['uuid']
+        library = input_args['lb']
+        rg = "@RG\tID:%s\tLB:%s\tPL:ILLUMINA\tSM:%s" % (uuid,
```

Comment:
Saw your comment on #151 and updated this. Thanks for pointing that out. It's a definite hack but... GD GATK, amirite?